tests/common/dualtor/dual_tor_io.py

```diff
@@ -390,10 +410,13 @@ def examine_each_packet(packets):
                     sent_payload = int(str(packet[scapyall.TCP].payload).replace('X',''))
                     sent_packets[sent_payload] = packet.time
                     continue
-                if packet[scapyall.Ether].src == self.received_pkt_src_mac:
+                if packet[scapyall.Ether].src in self.received_pkt_src_mac:
                     # This is a received packet.
                     received_time = packet.time
                     received_payload = int(str(packet[scapyall.TCP].payload).replace('X',''))
+                    if received_payload == prev_payload:
+                        duplicated_packets[received_payload] = packet.time
```

Comment:
if you continue to receive duplicate packets, then this logic will not give the actual count of duplicate packets. Suppose we receive 3 duplicate packets, I am assuming the received payload is going to be the same for all 3, then we just end up overwriting the same key and  your count for duplicate will be len(duplicated_packets) which is 1.

Reply:
I see the point. I was expecting only 2 duplicated packets (since server-to-t1 traffic is sent to both TORs, so one packet can come to T1 from two T0s.

For any weird reason, there can be more than two packets. I will update this accordingly.

Reply:
Updated now. `duplicated_packets_count` is incremented for every duplicated packet. Removed the dict() now, if we need some analysis on dropped packets, I can add this back later.